platform/src/core/utils/setAuthToken.js

```diff
@@ -0,0 +1,32 @@
+import axios from 'axios';
+import jwt_decode from 'jwt-decode';
+
+const setAuthToken = (token) => {
+    if (token) {
+        const decoded_token = jwt_decode(token);
+        const tenant = decoded_token.organization;
+
+        if (typeof tenant === 'undefined') {
+            // remove stored tokens to enable user to re-login
+            localStorage.removeItem('jwtToken');
+            localStorage.removeItem('currentUser');
+            throw new Error(
+                "This user's organization could not be determined, Please refresh to login again"
+            );
+        }
+
+        // Apply authorization token to every request if logged in
+        axios.defaults.headers.common['Authorization'] = token;
+        axios.interceptors.request.use((config) => {
+            const params = config.params || {};
+
+            return { ...config, params: { tenant, ...params } };
+        });
+    } else {
+        // Update auth header token to support data access for non-auth users
+        axios.defaults.headers.common[
+            'Authorization'
+        ] = `JWT ${process.env.REACT_APP_AUTHORIZATION_TOKEN}`;
+    }
+};
+export default setAuthToken;
```

Comment:
We are meant to use access tokens and not jwts I thought @FaithDaka

Reply:
But I thought JWTs are less secure because they can be decoded hence the "migration" to access tokens. As long as we can still pass jwts from the frontend, this risk will exist @Baalmart